<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToProjectDialog</class>
 <widget class="QDialog" name="AddToProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add to Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>How should each file be added?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ui_moveButton">
     <property name="text">
      <string>Move to the project folder</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ui_copyButton">
     <property name="text">
      <string>Make a copy and move the copy to the project folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ui_absRefButton">
     <property name="text">
      <string>Make a reference (absolute path)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ui_relRefButton">
     <property name="text">
      <string>Make a reference (relative path)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddToProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddToProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
